scripts/devops_tasks/Smoke-Tests-Release.ps1

```diff
@@ -0,0 +1,58 @@
+#Requires -Version 7.0
+
+Param(
+    [string] $serviceDirectory
+)
+
+$repoRoot = Resolve-Path "$PSScriptRoot/../../"
+
+Push-Location $repoRoot
```

Comment:
The way you are using Push-Location and Pop-Location. If anything goes wrong, you'll have a script that has changed your directory.